sdk/src/main/scala/com/horizen/account/mempool/MempoolMap.scala

```diff
@@ -122,33 +122,35 @@ class MempoolMap(
           val existingTxWithSameNonceIdOpt = nonExecTxsPerAccount.get(ethTransaction.getNonce)
           if (existingTxWithSameNonceIdOpt.isDefined) {
             val existingTxWithSameNonceId = existingTxWithSameNonceIdOpt.get
-            replaceTransaction(existingTxWithSameNonceId, ethTransaction, nonExecTxsPerAccount)
+            replaceTransaction(existingTxWithSameNonceId, ethTransaction, nonExecTxsPerAccount, isNotExec = true)
```

Comment:
Cosmetic: here and in other method signatures, use is made of  a boolean for distinguishing exec/nonexec. Consider using an Enumeration or a constant, which would render the caller code more readable

Reply:
Done